numpy/lib/function_base.py

```diff
@@ -4281,14 +4502,12 @@ def _percentile(a, q, axis=None, out=None,
     # avoid expensive reductions, relevant for arrays with < O(1000) elements
     if q.size < 10:
         for i in range(q.size):
-            if q[i] < 0. or q[i] > 100.:
-                raise ValueError("Percentiles must be in the range [0,100]")
-            q[i] /= 100.
+            if q[i] < 0. or q[i] > 1.:
+                raise ValueError("Quantiles must be in the range [0,1]")
     else:
         # faster than any()
-        if np.count_nonzero(q < 0.) or np.count_nonzero(q > 100.):
-            raise ValueError("Percentiles must be in the range [0,100]")
-        q /= 100.
+        if np.count_nonzero(q < 0.) or np.count_nonzero(q > 1.):
+            raise ValueError("Quantiles must be in the range [0,1]")
```

Comment:
This might confuse users who call `np.percentile`, but then get an error message mentioning quantile. But maybe that's ok